### PR TITLE
Adds a hotkey for place acid well

### PR DESCRIPTION
**Pull request by lorianss**
## About The Pull Request

Adds a hotkey for "Place acid well" - xeno 's ability

## Why It's Good For The Game

Easy access to the use of the ability 
Improving the quality of the game

## Changelog

:cl:
add: Xenomorphs can quickly build acid wells, as well as collapse the top panel with abilities (since all abilities can be used using keyboard shortcuts).
/:cl:

### CODE REVIEW

**Comment by TiviPlus** [comment]
Checking long list formatting
code/modules/mob/living/carbon/xenomorph/castes/shrike/abilities_shrike.dm:309:	keybinding_signals = list(
	KEYBINDING_NORMAL = COMSIG_XENOABILITY_PSYCHIC_CURE,
long list underindented, should be two tabs

**Comment by TiviPlus** [comment]
Also add a sensible default if you dont mind

File: code/modules/mob/living/carbon/xenomorph/castes/shrike/abilities_shrike.dm [hunk 1]
```diff
@@ -306,6 +306,9 @@
 	mechanics_text = "Place an acid well that can put out fires."
 	plasma_cost = 400
 	cooldown_timer = 2 MINUTES
+	keybinding_signals = list(
+	KEYBINDING_NORMAL = COMSIG_XENOABILITY_PSYCHIC_CURE,
```
Comment (Lumipharon) [inline code/modules/mob/living/carbon/xenomorph/castes/shrike/abilities_shrike.dm:310]: Shouldn't this be the new define?

**Review by Lumipharon — CHANGES_REQUESTED** [review]

File: code/datums/keybinding/xeno.dm [hunk 2]
```diff
@@ -40,6 +40,12 @@
 	description = "Builds whatever you’ve selected with (choose resin structure) on your tile."
 	keybind_signal = COMSIG_XENOABILITY_SECRETE_RESIN
 
+/datum/keybinding/xeno/place_acid_well
+	name = "place_acid_well"
+	full_name = "Place acid well"
+	description = "Builds acid well on your tile."
+	keybind_signal = COMSIG_XENOABILITY_PLACE_ACID_WELL
```
Comment (Lumipharon) [inline code/datums/keybinding/xeno.dm:47]:
```suggestion
	keybind_signal = COMSIG_XENOABILITY_PLACE_ACID_WELL
	hotkey_keys = list("T")
```
Add a keybind default here like this. Doesn't actually have to be T.

**Review by Lumipharon — APPROVED** [review]

**Pull request merged**